data/json/requirements/toolsets.json

```diff
@@ -24,7 +24,13 @@
     "id": "sewing_standard",
     "type": "requirement",
     "//": "Crafting or repair of fabric items",
-    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ] ] ]
+    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ], [ "sewing_machine", 1 ] ] ]
+  },
+  {
+    "id": "knitting_standard",
```

Comment:
I only added `sewing_machine` to `sewing_standard`, but it is already used for repairing of vehicle `roof_cloth` (see [data/json/vehicle_parts.json](https://github.com/CleverRaven/Cataclysm-DDA/blob/8a67f93818164def0f08fee422ef4e2b70eae648/data/json/vehicle_parts.json)).

`knitting_standard` is not used anywhere, so I will remove it.